just.config.js

```diff
@@ -22,8 +22,10 @@ task('storybook', () => parallel('lint/fix', 'storybook/serve'))
 task('storybook/serve', () => step('start-storybook -p 9009 -s public --quiet', { withWarn: true }))
 task('storybook/build', () => step('build-storybook -s public --quiet', { withWarn: true }))
 
+task('install', () => series('install/holoflows'))
 task('install/holoflows', async () => {
     const base = path.join(process.cwd(), 'node_modules/@holoflows')
+    await step('yarn upgrade @holoflows/kit')
```

Comment:
Hmm.. In old days we install @holoflows/kit by `yarn link`. That enable us to develop these two libraries in the same time.

But now @holoflows/kit is more stable than the old days, so month ago I make it as a dependency installed from git.

If kit gets more stable, we may install it directly from npm in the future.

Reply:
yeah a npm package can be helpful